LICENSE

```diff
@@ -0,0 +1 @@
+License to be determined.
```

Comment:
Should we use an MIT license? Should we attribute this to "Jeff Klukas" or to Simple? Should we move this to the BankSimple org?